### PR TITLE
First cut of SARS data cleaning + start of analysis

### PR DESCRIPTION
**Pull request by cm401**

### CODE REVIEW

File: .gitignore
Comment (thomrawson) [inline .gitignore:1]: Could we just ignore these in your git window instead? 
I can envision a world where one day we have a src with some data files and we wonder why it isn't committing!

Comment (cm401) [reply .gitignore:1]: Why would anyone want those though?

Comment (thomrawson) [reply .gitignore:1]: Take src/data_cleaning/data_dictionary.csv for example. With this commit, that file would no longer get pushed if it was a new task.

File: src/db_compilation/orderly.R [hunk 1]
```diff
@@ -337,7 +346,7 @@ if (pathogen == "LASSA") {
   outbreak_all  <- clean_dfs(outbreak_all, pathogen)
 }
 
-if (pathogen == "EBOLA") {
+if (pathogen %in% c("EBOLA","SARA")) {
```
Comment (thomrawson) [inline src/db_compilation/orderly.R:349]: Typo, think this should be SARS!

Comment (cm401) [reply src/db_compilation/orderly.R]: Thank you, that's fixed.

**Pull request closed without merging**